README.md

```diff
@@ -41,31 +33,22 @@ Short summary on functionality and used technologies.
 ---
 
 ## Minimal Path to Awesome
-
 - Clone this repository
 - Ensure that you are at the solution folder
 - in the command-line run:
   - **npm install**
-  - **gulp serve**
-
-> Include any additional steps as needed.
+  - **gulp bundle --ship**
+  - **gulp package-solution --ship**
+- The package will be available in the following path: \custom-welcome-banner\sharepoint\solution\custom-welcome-banner.sppkg
+- Add created package to your app catalog
+- Add the app to a SharePoint site and then add the web part to the page
 
 ## Features
-
-Description of the extension that expands upon high-level summary above.
-
-This extension illustrates the following concepts:
-
-- topic 1
-- topic 2
-- topic 3
-
-> Notice that better pictures and documentation will increase the sample usage and the value you are providing for others. Thanks for your submissions advance.
-
-> Share your web part with others through Microsoft 365 Patterns and Practices program to get visibility and exposure. More details on the community, open-source projects and other activities from http://aka.ms/m365pnp.
+This Web Part illustrates the following concepts on top of the SharePoint Framework:
+* Using the SPFx context to gather the current user's display name.
+* Language selection (English/French) through the property pane.
```

Comment:
You also have the account option. Maybe give more detail around it. Like it match the user profile language

Reply:
Updated.